### PR TITLE
Feature/headers usersettings video

### PR DESCRIPTION
**Pull request by andrisLaduzans**
#### headers
- all headers moved to view components , navigation headers no longer being used

#### video
- video view is now functional, but with no data.

#### user settings
- added another navigation stack 'InitialUserSettingsStack' which first sets language, then sets user type, and forwards to application after set user.

#### removes
- removed 'Also available in english' texts in settingsView and initial UserSelectView since in this version we are assuming that everything will be available in 2 languages

### CODE REVIEW

File: src/components/filters/Filters.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { TextToggleBtn } from '@components/buttons';
+import { LocalizationContext } from '../../localization/LocalizationContext';
+import { colors } from '@styles';
+
+interface FiltersProps {
```
Comment (a-sandris) [inline src/components/filters/Filters.tsx]: FilterProps would be better use of english

File: src/components/headers/SearchHeader.tsx [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 import { styles } from './styles';
-import { SharedStackNavList } from 'src/navigation/stacks/SharedStack';
+import { SharedStackParamsList } from 'src/navigation/stacks/SharedStack';
+import { StackNavigationProp } from '@react-navigation/stack';
 
-const SearchHeader: React.FC<SharedStackNavList<'Search'>> = ({ navigation }) => {
+interface Props {
+    navigation: StackNavigationProp<
+        SharedStackParamsList,
+        'Feed' | 'Favorites' | 'Article' | 'Search' | 'UserSettings'
```
Comment (a-sandris) [inline src/components/headers/SearchHeader.tsx]: extract this to type somewhere

File: src/localization/translations.ts [hunk 3]
```diff
@@ -21,6 +21,7 @@ const translations = new LocalizedStrings({
         TOMORROW: 'Rīt',
         THIS_WEEK: 'Šonedēļ',
         OTHERS: 'Cits',
+        ONLINE: 'Tiešaitē',
```
Comment (a-sandris) [inline src/localization/translations.ts]: typo

File: src/components/headers/SearchHeader.tsx (hunk 2 above)
Comment (a-sandris) [reply src/components/headers/SearchHeader.tsx]: or even enum

File: src/components/filters/Filters.tsx (hunk 1 above)
Comment (andrisLaduzans) [reply src/components/filters/Filters.tsx]: changed to FilterProps

File: src/components/headers/SearchHeader.tsx (hunk 2 above)
Comment (andrisLaduzans) [reply src/components/headers/SearchHeader.tsx]: Removed unnecesary enums 
SearchHeader to 'Search' only 
and Simple header to 'Favorites'|'Article'|'UserSettings'
added AnyType in domain/ which can be used as 'any' changed all 'any's to AnyType

File: src/localization/translations.ts (hunk 3 above)
Comment (andrisLaduzans) [reply src/localization/translations.ts]: fixed

**Pull request merged**